src/main/java/io/jenkins/pluginhealth/scoring/service/UpdateCenterService.java

```diff
@@ -0,0 +1,43 @@
+package io.jenkins.pluginhealth.scoring.service;
+
+import java.io.IOException;
+import java.net.URL;
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+
+@Service
+public class UpdateCenterService {
+    @Autowired
+    ObjectMapper objectMapper;
+    
+    PluginService pluginService;
+        
+    public UpdateCenterService(PluginService pluginService) {
+        this.pluginService = pluginService;
+    }
+
+    public void readUpdateCenter(String updateCenterURL) throws StreamReadException, DatabindException, IOException {
+        record UpdateCenterPlugin(String name, String scm, ZonedDateTime releaseTimestamp) {
+        }
+
+		record UpdateCenter(Map<String, UpdateCenterPlugin> plugins) {
+		}
+
+		URL jsonUrl = new URL(updateCenterURL);
+
+		UpdateCenter updateCenter = objectMapper.readValue(jsonUrl, UpdateCenter.class);
```

Comment:
I used the "readValue()" method as I was not able to find a "map...()" method for the objectMapper object. Please correct me if I'm wrong.